guides/common/modules/proc_creating-a-webhook.adoc

```diff
@@ -0,0 +1,43 @@
+[id="creating-a-webhook_{context}"]
+= Creating a Webhook
+
+.Procedure
+
+. In the {Project} web UI, click *Administer* > *Webhooks*.
+. Click *Create Webhook*.
+. Click *Subscribe to field* to reveal a list of available events.
+. Select an event from the list.
+. Enter a name.
+. Enter a target URL. When using the shellhooks plugin, the URL should be in the form of `https://{smartproxy.example.com}:{smartproxy_port}/shellhooks/my_script`.
+. Click *Template* to see the list of available templates.
+. Select a template from the list.
+. Enter a HTTP method.
+. Ensure that the *Enabled* flag is checked if you want to create active webhook.
+
+In the Credentials tab:
```

Comment:
We do not document by tab. 
Please rewrite with the step "Click the Credentials tab" and then integrate the rest of the steps as steps on their own. 
We write user-oriented not web ui oriented.